conda_lock/conda_lock.py

```diff
@@ -534,6 +555,11 @@ def main():
     multiple=True,
     help="path to a conda environment specification(s)",
 )
+@click.option(
+    "--filename-template",
```

Comment:
Perhaps `-o` and `--output` might be less surprising?

Reply:
I have no preference, I'll leave this up to the maintainers